docs/management_commands.rst

```diff
@@ -0,0 +1,74 @@
+Django Management Commands
+==========================
+
+Export Command
+--------------
+
+The ``export`` command allows you to export data from a specified Django model
+or a resource class. The exported data can be saved in different formats, such
+as CSV or XLSX.
+
+Usage
+-----
+
+.. code-block:: bash
+
+    python manage.py export <format> <resource> [--encoding ENCODING]
+
+- **format**: Specify the format in which the data should be exported. -
+- **resource**: Specify the resource or model to export. Accepts a resource class or a model class in dotted path format. - **--encoding** (optional): Specify the encoding (e.g., 'utf-8') to be used for the exported data.
+
+Example
+-------
+
+.. code-block:: bash
+
+    python manage.py export CSV auth.User
+
+This command will export the User model data in CSV format using utf-8
+encoding.
+
+Another example:
+
+.. code-block:: bash
+
+    python manage.py export XLSX mymodule.resources.MyResource
+
+This command will export the data from ``MyResource`` resource in XLSX format
+using utf-8 encoding.
+
+Import Command
+--------------
+
+The ``import`` command allows you to import data from a file into a specified
+Django model or a custom resource class.
+
+Usage
+-----
+
+.. code-block:: bash
+
+    python manage.py import <resource> <import_file_name> [--format FORMAT] [--encoding ENCODING] [--dry-run] [--raise-errors]
+
+- **resource**: The resource class or model class in dotted path format.
+- **import_file_name**: The file from which data is imported (``-`` can be used to indicate stdin).
+- **--format** (optional): Specify the format of the data to import. If not provided, it will be guessed from the mimetype.
+- **--encoding** (optional): Specify the character encoding of the data.
+- **--dry-run**: Perform a trial run without making changes.
+- **--raise-errors**: Raise any encountered errors during execution.
```

Comment:
> I notice that it seems to import fine but there is no indication of success. Perhaps a verbose flag could output what the result of the import was.

Good point. Updated so result is displayed even when dry-run is not used.

> I also tested --dry-run with 200 books and the output was 200 rows imported, which is correct but some of these were creates and some were updates. I think it would be ideal to have a breakdown of the results as we see in the Admin interface (i.e. num imported, skipped, updated etc)

Agree, added.